### PR TITLE
chore(.github): add logging code owner: observability dpe

### PR DESCRIPTION
**Pull request by nicoleczhu**
@tbpg is it possible to add me and obs team as codeowners to the logging samples? 
Thanks!

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -10,3 +10,4 @@
 *                               @googleapis/yoshi-go-admins
 
 /storage/        @GoogleCloudPlatform/storage-dpe @googleapis/yoshi-go-admins
+/logging/        @GoogleCloudPlatform/observability-devx @nicoleczhu
```
Comment (tbpg) [inline .github/CODEOWNERS:13]: Please add a newline at the end.

Comment (tbpg) [inline .github/CODEOWNERS:13]: Please include yoshi-go-admins, too.

**Review by tbpg — CHANGES_REQUESTED** [review]
Thank you!

File: .github/CODEOWNERS (hunk 1 above)
Comment (nicoleczhu) [reply .github/CODEOWNERS]: good catch and added!

Comment (nicoleczhu) [reply .github/CODEOWNERS]: added!

**Review by tbpg — APPROVED** [review]
Thanks!

**Pull request merged**